docs/core/deploying/native-aot/ios-like-platforms/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Native AOT support for iOS-like platforms overview
+description: Learn how Native AOT supports iOS-like platforms
+author: ivanpovazan
+ms.author: ivanpovazan
+ms.date: 11/21/2024
+---
+
+# Native AOT support for iOS-like platforms
+
+Starting from .NET 8, Native AOT supports targeting iOS-like platforms. The term *iOS-like platforms* refers to Apple platforms that use similar APIs such as: iOS, MacCatalyst and tvOS.
+
+In the first release, the support was experimental, while in .NET 9 these platforms received full support with Native AOT.
```

Comment:
I do not think we need to be going over the history and that we have shipped it as an experiment for one release. We should just say that this is supported starting .NET 9.